src/trains/handlers/trainHandler.go

```diff
@@ -0,0 +1,101 @@
+package handlers
```

Comment:
trains ... handlers .... trainHandlre.go ? 

I suppose it should be 
service/trains/trains.go